la.ogri.strongbox.yml

```diff
@@ -0,0 +1,56 @@
+app-id: la.ogri.strongbox
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
+    - org.freedesktop.Sdk.Extension.openjdk11
+
+command: strongbox
+
+finish-args:
+    - "--socket=x11"
+    - "--share=network"
+    - "--device=dri"
+    - "--filesystem=host"
+
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+        - /usr/lib/sdk/openjdk11/install.sh
+
+    - name: strongbox
+      buildsystem: simple
+      sources:
+        - type: file
+          path: app.jar
```

Comment:
```suggestion
```

that's invalid, and the reason the build fail.

Reply:
yes, I can see that. How/why is it invalid?